### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by SunSetPilot**

### CODE REVIEW

**Review by whc1202 — COMMENTED** [review]
fantastic!

File: mcr.cpp [hunk 1]
```diff
@@ -1,51 +1,61 @@
...
+            cout << "Player 1: ";
+            character = player1;
+        }
+        else {
+            cout << "Player 2: ";
+            character = player2;
+        }
+        cout << "Which cell to mark? i:[1..3], j:[1..3]: ";
+        cin >> i >> j;
+        while ((i < 1 || i > 3 || j < 1 || j > 3) || (game[i - 1][j - 1] == 'X' or game[i - 1][j - 1] == 'O')) {
```
Comment (n1cezy) [inline mcr.cpp:40]: You should use '||' instead of 'or'

File: mcr.cpp [hunk 2]
```diff
@@ -1,51 +1,61 @@
...
+    int i, j;
+    char game[3][3] = {{' ', ' ' , ' '},
+                       {' ', ' ' , ' '},
+                       {' ', ' ' , ' '} }; // Tic-tac-toe
+    bool player2_turn = true; // false for player 1's player2_turn, true for player 2's player2_turn. Player 1 first.
+    int n;
+    const char player1 = 'X';
+    const char player2 = 'O';
+    cout << player1 << " = Player 1" << endl << player2 <<" = Player 2" << endl;
+    for (n = 0; n < 9; n++){
```
[the remark this answers is not in the document]
Comment (TianTianSuper) [reply mcr.cpp:27]: Oh I see the definition in line 18

[the remark this answers is not in the document]
Comment (SunSetPilot) [reply mcr.cpp:27]: I will put the definition closer to this place

**Pull request merged**